plugins/argentina-afip-invoices/src/classes/Argentinacae/claves/cwkey.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
why is there a private key included? are you sure it should be here?

Reply:
Hi @keksa this private key is used when customer decides to use the "debug function" included in the UCRM settings... With Debug, the plugin uses this private key to get "test results" instead to requiere a valid new certificate....